public/page2.html

```diff
@@ -0,0 +1,94 @@
+<title></title></head>
+<style type="text/css">
+body{
+background-color: #ecf0f1;
+font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+font-style: normal;
+  font-variant: normal;
+  font-weight: 500;
+  line-height: 26.4px;
+}
+.navbar-inverse .navbar-nav > .active > a, .navbar-default .navbar-nav > .active > a:hover, .navbar-default .navbar-nav > .active > a:focus {
+  background-color: #16a085;}
+  .navbar-inverse{
+  background-color: #1abc9c;
+    } .container{
+  width: 100%;
+  }.btn-default.btn-on.active{background-color: #5BB75B;color: white;}
+.btn-default.btn-off.active{background-color: #DA4F49;color: white;}
+</style> 
+   <body>
+      <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+        <div class="navbar-header">
+             <a class="navbar-brand" href="#"></a>
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+            </button>
+       </div>
+         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+           <ul class="nav navbar-nav navbar-right">
+              <li class="dropdown">
+               <a href="#" class="dropdown-toggle" data-toggle="dropdown" style="color: #fff">Rupam Verma<b class="caret"></b></a>
+                <ul class="dropdown-menu">
+                   <li class="letter"><a  href="#"></a>R</a></li>
+                   <li class="leter"><a href="#"></a>Rupam Verma</a></li>
+                   <li class="leter"><a href="#"></a>Mahi.roops@gmail.com</a></li>
+                </ul>
+             </li>
+          </ul>
+      </div>
+  </nav>
+      <div class="container">
+         <div class="row">
+           <div class="col-xs-12" style="background-color: #34495e";>
+            <h3 class="header">Menu</h3>
+      </div>
+          <div id="icon" class="icon-bar">
+            <a  href="index_task.html"><span class="icon-circle"></span></a> 
+              <a class="active" href="#page2"><span class="icon-toggle-off"></span></i></a> 
+               <a href="page3.html"><span class="icon-checklist"></span></a></div>
+                 <h3 class="toggle" style="text-align: center;margin-top: 15%">Toggle Switch</h3>
+                   <div class="btn-group" id="status" data-toggle="buttons">
+                     <label class="btn btn-default btn-on btn-sm active">
+                       <input type="radio" value="1" name="multifeatured_module[module_id][status]" checked="checked">C</label>
+                         <label class="btn btn-default btn-off btn-sm ">
+                           <input type="radio" value="0" name="multifeatured_module[module_id][status]">F</label>
+                      </div>
+                  <br>

```

Comment:
@rupamverma Why not `margin`  to add space between 2 containers instead of `<br>` tag.